sunpy/time/tests/test_unixtai58.py

```diff
@@ -0,0 +1,50 @@
+import pytest
+
+from astropy.time import Time, TimeUnixTai
+from astropy.time.formats import erfa
+
+# This registers the TimeTAICDS format with astropy
+from sunpy.time import TimeUnixTai58  # NOQA
```

Comment:
```suggestion
import sunpy.time  # NOQA
```
Should work too, and is a good test that we're importing TimeUnixTai58 into the `sunpy.time` namespace? This isn't a blocking suggestion though.